mbedtls/src/x509/certificate.rs

```diff
@@ -253,6 +263,27 @@ impl Certificate {
         }
         result.map(|_| ())
     }
+
+    pub fn verify(
+        chain: &MbedtlsList<Certificate>,
+        trust_ca: &MbedtlsList<Certificate>,
+        err_info: Option<&mut String>,
+    ) -> Result<()> {
+        let none: Option<Box<dyn VerifyCallback + 'static>> = None;
+        Self::verify_ex(chain, trust_ca, err_info, none)
```

Comment:
```suggestion
        Self::verify_ex(chain, trust_ca, err_info, None::<&dyn VerifyCallback>)
```

Reply:
Oh nice! I didn't know about that syntax